packages/forma-36-react-components/src/components/Notification/NotificationItemContainer.tsx

```diff
@@ -3,7 +3,7 @@ import AnimateHeight from 'react-animate-height';
 import NotificationItem, { NotificationItemProps } from './NotificationItem';
 
 export interface NotificationItemContainerProps extends NotificationItemProps {
-  duration: number;
+  duration: number | undefined;
```

Comment:
Not sure if this is a terrible idea or not, but it got around TypeScript complaining

Reply:
@m10l what do you want to express here? that `duration` can be optional? In that case you can rewrite the interface like: 
```suggestion
  duration?: number;
```